### PR TITLE
12 project setup

### LINKED ISSUE #12: Project setup
Set up the following to be tested at client system.

python project with dependencies.txt
base "hello world" module that has function that returns "hello world"
unit test called "test_hello.py", in a "test" folder to verify the above

### PR DESCRIPTION
**Pull request by rnllv**
Fixes #12
Facing import issue when trying to access src from a different directory. Working to fix the same.

image

### CODE REVIEW

**Comment by IanMayo** [comment]
note: from the root I was able to run `python -m src.hello`

**Comment by rnllv** [comment]
Import issue resolved now. Updated `README.md` contains the details.

**Comment by rnllv** [comment]
image

File: legacyman_parser/hello.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import sys
+from os.path import dirname
```
Comment (IanMayo) [inline legacyman_parser/hello.py]: it looks like this import isn't used

**Comment by IanMayo** [comment]
I'd like us to support the `unittest discover` capability, please @rnllv

**Comment by rnllv** [comment]
Done. I just edited the main function to return string, instead of printing it.

image

File: README.md [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+
+Install these automatically using the below `pip` command from the same directory.
+
+```
+pip install -r requirements.txt
+```
+
+## Running tests
+
+Execute the following command from inside the `test` directory to run the tests.
```
Comment (IanMayo) [inline README.md]: could you see if this will run from `root`, please?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,41 @@
...
+
+## Technologies
+
+- python >= 3.9
+
+## Initial Setup
+
+### Install dependencies
+
+Python dependencies are specified in `requiremnts.txt` in the root directory.
```
Comment (IanMayo) [inline README.md]: looks like a typo on `requiremnts`

**Comment by rnllv** [comment]
Commands to be executed from root directory.
image

**Comment by rnllv** [comment]
Unwanted imports removed.

**Review by IanMayo — APPROVED** [review]

**Pull request merged**